From 70d1b6ba9db4ecdaca10466cfbb7f1264aca9a92 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 5 Dec 2016 17:46:17 +0800
Subject: [PATCH 1370/5242] MLK-13607-2 video: mxsfb: add new compatible entry
 for lcdif on imx7ulp

commit  f089a982cd8da122db7b7b29b5b0bb24e055b824 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the compatible entry specific for imx7ulp platform in lcdif
driver.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit f0e73404a8177c9627d04300fc421684d22fa582)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 73bcf14..533612f 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1361,6 +1361,7 @@ static int mxsfb_unmap_videomem(struct fb_info *fbi)
 static const struct of_device_id mxsfb_dt_ids[] = {
 	{ .compatible = "fsl,imx23-lcdif", .data = &mxsfb_devtype[0], },
 	{ .compatible = "fsl,imx28-lcdif", .data = &mxsfb_devtype[1], },
+	{ .compatible = "fsl,imx7ulp-lcdif", .data = &mxsfb_devtype[1], },
 	{ /* sentinel */ }
 };
 MODULE_DEVICE_TABLE(of, mxsfb_dt_ids);
-- 
1.7.9.5

